community/asciinema/APKBUILD

```diff
@@ -1,32 +1,25 @@
 # Contributor: Fabian Affolter <fabian@affolter-engineering.ch>
 # Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
 pkgname=asciinema
-pkgver=1.2.0
+pkgver=1.4.0
 pkgrel=0
 pkgdesc="A Command line recorder for asciinema.org service"
 url="https://github.com/asciinema/asciinema/"
-arch="all !aarch64 !ppc64le !s390x"
+arch="all !aarch64 !s390x"
 license="GPLv3"
-depends="curl"
-depends_dev=""
-makedepends="go python2"
-subpackages="$pkgname-doc"
-options="!strip"
+depends="ncurses"
+makedepends="python3-dev py-setuptools"
```

Comment:
py-setuptools is only for python2. There's no py3 variant, because setuptools is included in stdlib of Python 3. So you should omit it.